projects/hangman/hangman_functions.py

```diff
@@ -74,22 +76,23 @@ def greeting_function():
     """
     flag = True
     print('\n')
-    print('                                                       *** WELCOME TO HANGMAN GAME ***')
+    termcolor.cprint('{:>100}'.format('*** WELCOME TO HANGMAN GAME ***'), 'yellow', attrs=['bold'])
     print('\n')
-    print('                                              <><><> LETS UNDERSTAND HOW TO PLAY THE GAME <><><>')
+    termcolor.cprint('{:>110}'.format('<><><> LETS UNDERSTAND HOW TO PLAY THE GAME <><><>'), 'yellow', attrs=['bold'])
```

Comment:
when i executed via command prompt or powershell then i get this 
![image](https://user-images.githubusercontent.com/12052941/105227343-794d5a80-5b61-11eb-8388-1b519593b6b0.png)